.github/CHANGELOG.md

```diff
@@ -28,6 +28,10 @@
   decompose a Hamiltonian into a linear combination of Pauli operators.
   [(#671)](https://github.com/XanaduAI/pennylane/pull/671)
 
+* Adds an `ApproxTimeEvolution` template to the PennyLane templates module, which 
```

Comment:
@ixfoduap I agree with your general sentiment! In this particular case, I think either @co9olguy or @mariaschuld (?) suggested `TrotterTimeEvolution`, since there is the potential to add templates for exact time evolution down the line

Reply:
yes, very likely that at some point we'll implement `H-->exp(itH)` at some stage